### PR TITLE
Fix/testing env http status forbidden

### PR DESCRIPTION
**Pull request by felder-cl**
docker: Add suggestion for HTTP Forbidden response

Suggest double checking URL or disabling HTTP's Allow Origins for
testing when getting HTTP Status Forbidden responses.

Also, make slight simplification in codebase.

https://www.pivotaltracker.com/story/show/170472180

### CODE REVIEW

**Comment by HenryNguyen5** [comment]
What we could do is set `ALLOW_ORIGINS` to "*" for all of our docker commands. You'd have to do the follwoing: 
- remove this line https://github.com/smartcontractkit/chainlink/blob/develop/tools/docker/docker-compose.integration.yaml#L75
- add the env here https://github.com/smartcontractkit/chainlink/blob/develop/tools/docker/.env
- pass it in the base config here: https://github.com/smartcontractkit/chainlink/blob/develop/tools/docker/docker-compose.yaml#L12

**Comment by j16r** [comment]
Sounds good @HenryNguyen5, want to add this @felder-cl ?

**Comment by felder-cl** [comment]
Unfortunately I disagree, though a good suggestion.  The problem appears to be more a visibility issue than anything else.  '127.0.0.1' is denied while 'localhost' is allowed.

Also, the problem with adding the configuration variable is a bit nuanced.  Its a docker environment specific solution.  If I run into a problem testing Chainlink outside of the docker environment, is it due to configuration?  What's the correct configuration for testing?  It makes answering these questions more difficult.

Alternative suggestions could be:

  - updating the default configuration to include ip addresses for 'allow origin'
  - make 'allow origin' lenient when CHAINLINK_DEV is enabled
  - or increase visibility in the logs or elsewhere about what the origin of the problem is

File: core/web/router.go [hunk 1]
```diff
@@ -363,13 +363,16 @@ func uiCorsHandler(config orm.ConfigReader) gin.HandlerFunc {
...
-		if len(allowOrigins) > 0 {
-			c.AllowOrigins = allowOrigins
+	} else if allowOrigins := strings.Split(config.AllowOrigins(), ","); len(allowOrigins) > 0 {
+		c.AllowOrigins = allowOrigins
+	}
+	handler := cors.New(c)
+	return func(c *gin.Context) {
+		handler(c)
+		if status := c.Writer.Status(); status >= 400 && status < 500 {
+			logger.Info("request denied by Cross-Origin Resource Sharing (CORS) policy")
```
Comment (j16r) [inline core/web/router.go]: Isn't a CORS policy enforced by the browser?

Comment (felder-cl) [reply core/web/router.go]: Good point

**Comment by felder-cl** [comment]
To enhance visibility, I've created a follow up [task](https://www.pivotaltracker.com/story/show/170624717) for the frontend.

What's the stat on the PR?

**Pull request merged**